cache-config/testing/ort-tests/t3c-os-hostname-short_test.go

```diff
@@ -0,0 +1,129 @@
+package orttest
+
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/apache/trafficcontrol/cache-config/t3cutil"
+	"github.com/apache/trafficcontrol/cache-config/testing/ort-tests/tcdata"
+)
+
+func TestT3cApplyOSHostnameShort(t *testing.T) {
+	tcd.WithObjs(t, []tcdata.TCObj{
+		tcdata.CDNs, tcdata.Types, tcdata.Tenants, tcdata.Parameters,
+		tcdata.Profiles, tcdata.ProfileParameters, tcdata.Statuses,
+		tcdata.Divisions, tcdata.Regions, tcdata.PhysLocations,
+		tcdata.CacheGroups, tcdata.Servers, tcdata.Topologies,
+		tcdata.DeliveryServices}, func() {
+		doTestT3cApplyOSHostnameShort(t)
+	})
+}
+
+func doTestT3cApplyOSHostnameShort(t *testing.T) {
+
+	t.Logf("DEBUG TestT3cApplyOSHostnameShort starting")
```

Comment:
Was this left in from debugging?

Reply:
They were intentional, I think they're useful for debugging the test if it breaks. But you're right, I should remove the "DEBUG" text and make them conform to the existing t3c test idioms

Reply:
Done